walk-in-interview/src/main/java/com/google/configuration/TestFireStoreConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package com.google.configuration;
+
+/** Class that specify the configuration details for firestore during testing phase. */
+public final class TestFireStoreConfiguration implements FireStoreConfiguration {
+    private static TestFireStoreConfiguration developmentFireStoreConfiguration = null;
+
+    private static final String PROJECT_ID = "";
+
+    private TestFireStoreConfiguration() {}
+
+    public static TestFireStoreConfiguration getFireStoreConfiguration() {
+        if (developmentFireStoreConfiguration == null) {
```

Comment:
Optionally, if it's not implemented at this stage, you can throw not implemented exception. At least you can pin point why the test fail, such as due to missing config implementation.